artemis-server/src/main/java/org/apache/activemq/artemis/core/server/ActiveMQServerLogger.java

```diff
@@ -309,6 +309,12 @@ void slowConsumerDetected(String sessionID,
    @Message(id = 221052, value = "trying to deploy topic {0}", format = Message.Format.MESSAGE_FORMAT)
    void deployTopic(SimpleString topicName);
 
+   @LogMessage(level = Logger.Level.INFO)
```

Comment:
What is the reason for adding a new message here (with different ID) vs just adding the change to the existing message?